### DISCUSSION TITLE
LOCAL STORAGE!??? ARE YOU KIDDING ME???? Of course not.

### DISCUSSION THREAD

**Discussion by unityconstruct** [kanriapp/kanri#549] [General]
# Simple. Local. Nice.
Not everyone wants to fork over all their personal data to the spooks at BigData/BigTech.

I've been running the Atlassian Stack on '$10 licenses for a decade and was floored with their betrayal of the open source community and pushing 'all things cloud'. Cloud is great for publishing, but for BizOps & MyOps I shake my head suffering through security training when the entire Corp dataset is whimsically stored on 3rd Party Hardware - particularly with the likes of M$/Amazon _(both unholy moon-children spawned straight from the loins of the Lower Astral Realms)_.

JIRA's Kanban(if you so desire) is uber configurable and has hooks and applinks galore, but for day-2-day mundane stuff, I just want to move as fast as I type & don't need 40 fields of metadata & workflows to close stuff.

Kanri.. obscenely barren. But the add of custom background with BLUR & BRIGHTNESS (something you don't lust for until you have it) transform barren to aesthetically beautiful. With local datastor, the app moves as fast as you do... JIRA has a gorillion callbacks and DB queries.... and when you just want to make line-items for "cut yard, take out trash, fix leaky faucet" all that stuff just gets in the way and 3 min of notes takes 10 - forgetting the last 5 things while jacking with drop downs and combo boxes.

OK.. that's my feedback.. when I saw the background feature, had to login and drop this novel.
Nice Work!
~UC

**Comment by trobonox** [comment]
Now that was very entertaining to read! :)
Thanks for your feedback and I'm glad you're enjoying Kanri. As a previous user of self-hosted Atlassian software and a general advocate for keeping your data offline when possible, you perfectly understood my aim with Kanri. 
Sadly, development has been stalled quite a bit in the last few months, but hopefully I can get back on track soon while still maintaining the simplicity everyone appreciates.

I noticed you just joined the Discord server (or at least I believe so), if you have any more inquiries feel free to ask me anything there and I will reply a lot faster than here on GitHub. Even if development is slow, I want the community to know that I'm still here and care about the project.
